src/ch05-02-example-structs.md

```diff
@@ -225,42 +242,44 @@ struct Rectangle {
 fn main() {
     let rect1 = Rectangle { width: 30, height: 50 };
 
-    println!("rect1 is {:?}", rect1);
+    println!("rect1 to {}", rect1);
 }
 ```
 
-<span class="caption">Listing 5-12: Adding the annotation to derive the `Debug`
-trait and printing the `Rectangle` instance using debug formatting</span>
+<span class="caption">Listing 5-12: Dodanie adnotacji derywacji cechy `Debug`
+i wyświetlanie instancji `Rectangle` formatowaniem przeznaczonym do celów debugowania</span> 
 
-Now when we run the program, we won’t get any errors, and we’ll see the
-following output:
+Teraz kiedy uruchomimy program nie wyskoczy nam żaden błąd, a naszym oczom
+ukaże się poniższy tekst:
 
 ```text
-rect1 is Rectangle { width: 30, height: 50 }
+rect1 to Rectangle { width: 30, height: 50 }
 ```
 
-Nice! It’s not the prettiest output, but it shows the values of all the fields
-for this instance, which would definitely help during debugging. When we have
-larger structs, it’s useful to have output that’s a bit easier to read; in
-those cases, we can use `{:#?}` instead of `{:?}` in the `println!` string.
-When we use the `{:#?}` style in the example, the output will look like this:
+No nieźle! Nie jest to może najpiękniejsza reprezentacja, ale spełnia swoje zadanie i
+pokazuje wartości wszystkich atrybutów tej instancji,
+co zdecydowanie by pomogło gdybyśmy polowali na bugi.
+Kiedy w grę wchodzą większe struktury miło byłoby też mieć troszkę czytelniejszy wydruk;
+w takich sytuacjach możemy użyć `{:#?}` zamiast `{:?}` w makrze `println!`.
+Użycie stylu `{:#?}` w tym przypadku wyglądało będzie tak:
 
 ```text
-rect1 is Rectangle {
+rect1 to Rectangle {
     width: 30,
     height: 50
 }
 ```
 
-Rust has provided a number of traits for us to use with the `derive` annotation
-that can add useful behavior to our custom types. Those traits and their
-behaviors are listed in Appendix C. We’ll cover how to implement these traits
-with custom behavior as well as how to create your own traits in Chapter 10.
+Rust oddaje nam do użytku cały szereg cech, które możemy użyć wspólnie z adnotacją `derive`,
```

Comment:
po cech dodałbym w nawiasie ang. traits

Reply:
rozumiem zamiar, ale nie jest to pierwszy raz kiedy ta terminologia jest używana w samej książce, ani w samym tym pliku/podrozdziale

Reply:
w takim razie ok
tu też inna sugestia, może zaryzykować:
które możemy użyć wspólnie z adnotacją `derive` -> które możemy nadać za pomocą adnotacji `derive`

Reply:
`które możemy nadać za pomocą adnotacji derive`
brzmi lepiej, zmienię